kyaml/kio/filters/fmtr.go

```diff
@@ -78,7 +79,12 @@ func (f FormatFilter) Filter(slice []*yaml.RNode) ([]*yaml.RNode, error) {
 			continue
 		}
 		kind, apiVersion := kindNode.YNode().Value, apiVersionNode.YNode().Value
-		s := openapi.SchemaForResourceType(yaml.TypeMeta{APIVersion: apiVersion, Kind: kind})
+		var s *openapi.ResourceSchema
+		if f.UseSchema {
+			s = openapi.SchemaForResourceType(yaml.TypeMeta{APIVersion: apiVersion, Kind: kind})
+		} else {
```

Comment:
fyi -- s is nil by default so this is a no-op, but is useful for code-coverage metrics (explicitly shows that s = nil is a tested case)